.gitignore

```diff
@@ -1,5 +1,32 @@
+# General
```

Comment:
Not sure why this file is included in the PR.

Reply:
Me neither, will remove this